src/09-word-problems.js

```diff
@@ -152,12 +217,28 @@ function gradeAssignments(assignments) {}
     createLineOrder(people);
     //> [ "Ray Anderson", "America Marsh", "Wade Carson", "Patience Patel" ]
  */
-function createLineOrder(people) {}
+function createLineOrder(people) {
+  let firstClass = [];
+  let coach = [];
+  let finalLine = [];
+
```

Comment:
Can we just talk about how much I love how you named these variables? Nothing makes me happier than variables that are appropriately named and this made my programming heart warm. 🔥 Nice work!